free-2.md

```diff
@@ -1,51 +1,65 @@
-# Free project 2
+# Free project 2: Automated Excel spreadsheet population
 
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project would serve people working in consulting, finance or accounting who want to automate or speed up the process of data retrieval and population of Excel spreadsheets. 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+People working in these fields often have to parse large amount of information, often in enormous PDF files. These documents like the 10-K contain data in tables that have to be manually copied and pasted into Excel spreadsheets with relationships and equations between cells systematically entered. This is often quite tedious and repetitive work - suggesting an opportunity for a DSL. With this DSL, certain simple equations could be simplified to commands, and strung together with the fluency of a programming language. Ideally, the use of this DSL would free up time and resources for people to devote on less mind-numbing activities. 
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+It would be able to automate some of this tedious work and improve productivity. 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+It could potentially be a very useful DSL. I came across this idea after conversations with peers working in economic consulting and my personal experience in Econ classes like Intro to Accounting. 
 
 ### Domain
 _Describe the project's domain in five words._
 
+Automated Excel spreadsheet population
 
 ### Interface (syntax)
```

Comment:
What does the syntax of the language look like? Could we see a small sample of a snippet of "code" in this language?